packages/flutter/test/cupertino/action_sheet_test.dart

```diff
@@ -575,7 +675,7 @@ void main() {
     await tester.tap(find.text('Go'));
     await tester.pump();
 
-    expect(findScrollableActionsSectionRenderBox(tester).size.height, moreOrLessEquals(112.3));
+    expect(findScrollableActionsSectionRenderBox(tester).size.height, moreOrLessEquals(84.0));
```

Comment:
why this change?

Reply:
same for other in the file

Reply:
Let me revert this change and propose it in a separate PR.

Reply:
I've reverted these changes (and changed the minimum height from 84.0 to 84.3). Now all previous tests pass without any adjustment!